ww3tools/SatGlobal_Altimeter.py

```diff
@@ -0,0 +1,297 @@
+#Description:
+
+#The provided Python script (SatGlobal_Altimeter_new.py) is designed to process satellite data based on the input parameters you provide.
+#The batch script sets these parameters dynamically and submits the job for processing. Make sure to modify the batch script
+#to match your specific requirements, such as the satellite name, time-averaging settings, and paths to input and output directories.
+
+#In the batch script one should define:
+#Set your desired start and end dates using the start_date and end_date variables.
+#Specify the input directory (input_directory) where your satellite data files are located.
+#Specify the output directory (output_directory) where you want to save the processed data.
+#Set the satellite name (satellite_name) and other time-averaging parameters (time_averaging, averaging_time) as needed.
+
+
+
+#In the foloowing code, parse_args() defines the expected command-line arguments for the script and their properties, such as whether they are required,
+#their data types, default values, and help messages. When the script is executed, argparse processes the command-line arguments, 
+#and you can access their values to configure the behavior of your script.
+#Users running the script would need to provide the path to this file as a command-line argument when they execute the script. 
+#The location of the input file would depend on the user's setup and the file structure of the satellite data they are working with.
+
+#The script defines several command-line arguments that users can provide when they run the script:
+#--time-averaging: This is a flag argument. If provided, it sets a Boolean value to True, indicating that time averaging should be enabled.
+#--duration: This argument allows users to specify the time averaging duration in seconds. It has a default value of 600 seconds (10 minutes).
+#--start-date and --end-date: These required arguments expect users to provide start and end date/time values in the format "YYYY-MM-DD HH:MM:SS."
+#-i or --input: This required argument expects users to provide the path to an input directory. It is likely the directory where satellite data files are located.
```

Comment:
@GhazalMohammadpour-NOAA  Your comment here says that the input directory is "likely" the directory where the satelite files are located.  Would this program work if the input directory was not the directory where the satellite files are located?

Reply:
I recommend removing the word "likely" and simply just saying it is the directory where the satellite files are located.